docker/integration_tests/baseline_tests.sh

```diff
@@ -82,7 +82,30 @@ fi
 rm ~/.ZAP_D/config.xml
 
 echo
-echo "TEST Baseline 4 (new vs old) - we expect _some_ differences and this will not fail the whole script"
+echo "Baseline test 4 (vs example.com with INFO/WARN/FAIL and OUTOFSCOPE set via URL)"
+/zap/zap-baseline.py -t https://www.example.com/ --auto -u https://raw.githubusercontent.com/zaproxy/zaproxy/main/docker/integration_tests/configs/baseline3.conf > /zap/wrk/output/baseline4.out
```

Comment:
We don't need to specify `--auto` now right?

Reply:
Good point, let me try that - it should work and if so I can remove that...

Reply:
Yep, all worked, so removed them :)